realizations/README.md

```diff
@@ -0,0 +1,7 @@
+## Model build/run
+
```

Comment:
Maybe add text to say "Example realization files are provided in this directory."